pkg/ebpf/c/common/filtering.h

```diff
@@ -214,13 +202,7 @@ statfunc u64 compute_scopes(program_data_t *p)
         return 0;
 
     proc_info_t *proc_info = p->proc_info;
-
     policies_config_t *policies_cfg = get_policies_config(p);
```

Comment:
since `get_policies_config()` function is now one line, I think it will be simpler to just put it inline